scripts/check-packages.js

```diff
@@ -0,0 +1,142 @@
+const fs = require('fs')
+const path = require('path')
+const glob = require('glob')
+const { printLog, printError, runMain } = require('./lib/execution-utils')
+const { command } = require('./lib/command')
+
+const PACKAGE_OPTIONS = {
+  'packages/worker': {
+    builtWithTypeScript: false,
+    extraFiles: [
+      // This file is not built with TypeScript and not listed by package.json entry points, so we
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** I have struggled a bit to understand why only this file was needed 😅‏
```suggestion
      // This file is not built with TypeScript and not listed by string/package.json entry points, so we
```